spec/requests/api/v1/user_runs_request_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+describe 'users runs API' do
+  before :each do
+    User.destroy_all
+    Run.destroy_all
+  end
+
+  it "can find a user's runs" do
```

Comment:
This spec checks to see if we can see a list of a specific user's runs with this endpoint.